lib/linguist/heuristics.yml

```diff
@@ -149,6 +149,12 @@ disambiguations:
     named_pattern: objectivec
   - language: C++
     named_pattern: cpp
+- extensions: ['.ice']
+  rules:
+  - language: Slice
+    pattern: '^\s*(#\s*(include|if[n]def|pragma)|module\s+[A-Za-z][_A-Za-z0-9]*)'
+  - language: JSON
+    pattern: '/\A\s*[{\[]/'
```

Comment:
I'm sorry for digging this out, but I'm not sure if such patter with 2 slashes works with [`Regexp.new`](https://github.com/github/linguist/pull/4087/files#diff-e56a8d50b1a75fc4212ffb3504faa007R79).

AFAIK it should not match, here is [REPL example](https://repl.it/repls/GeneralResponsibleDesign) where it indeed does not.

Saw @Alhadis https://github.com/github/linguist/pull/4243#issuecomment-423562417 with pattern suggestion and wanted to verify my understanding first here, before opening a new PR with removal of the slashes.

WDYT?

Reply:
Yeah, you're right. The JSON detection doesn't seem to be working right. I guess a new issue should be opened?

Reply:
Oops, you're right. Yes, those should be removed.

Reply:
@bzz Go ahead for the pull request!